.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Bug report
+about: Help us track down bugs in JuMP
+
+---
+
+Welcome to JuMP! The JuMP developers use the GitHub issue tracker for bug reports and feature requests only. Please read the following before posting a new issue:
```

Comment:
Given the split between bug report and feature request, maybe:

"Welcome to JuMP! Please read the following before posting a new bug report:"